lib/geo_faker/multi_polygon.rb

```diff
@@ -0,0 +1,45 @@
+module GeoFaker
+  class MultiPolygon
+    attr_reader :coordinates
+
+    def initialize(coordinates)
+      @coordinates = coordinates
+    end
+
+    def self.from_geojson(geojson)
+      case geojson.fetch('type')
+      when 'MultiPolygon'
+        MultiPolygon.new(geojson.fetch('coordinates'))
+      when 'Polygon'
+        MultiPolygon.new([geojson.fetch('coordinates')])
+      end
+    end
+
+    def contains_point(point)
+      coordinates.any? { |polygon| point_in_poly(polygon[0], point) }
```

Comment:
The name `coordinates` is a bit misleading, as it's an array of polygones?

Reply:
Yeah, I know. This way it is in line with the GeoJSON format, but I already thought about renaming it to `polygons`.